s3config-test.yaml

```diff
@@ -19,6 +19,9 @@
 
 S3Config_Sections: [S3_SERVER_CONFIG, S3_AUTH_CONFIG, S3_MOTR_CONFIG, S3_THIRDPARTY_CONFIG]
 S3_SERVER_CONFIG:                                     # Section for S3 Server
+   S3_RANGED_READ_ENABLED: true                         # enable ranged reads
+   S3_READ_MD5_CHECK_ENABLED: false                     # enable data integrity checksum check on S3 GET
```

Comment:
Shouldn't the default be true?

Reply:
Yes, I'd have to disable a number of UTs and STs for ranged read to make this work, and it would take more time. Let's see what is the preference of the team.